task-docs/init/prompts.md

```diff
@@ -0,0 +1,28 @@
+First draft for init prompts
+```
+> Init will create the folder structure for a new Origami component ...
+? Name: (required)
+? Description: (required)
+? Keywords: ([])
+? Support email: (origami.support@ft.com)
+? Support slack: (#ft-origami)
+? component status: (experimental)
+? brands: (master)
```

Comment:
it would be lovely if the two of these (`component status`, `brands`) were expandables, lists or tab-completable entries.